### ISSUE TITLE
Go Swift client

### ISSUE THREAD

**Issue by mlanner** [ncw/swift#64]
Not so much an issue, but more of a question/request:

Have you ever considered making this into a full Go Swift Client, as opposed to "just" a library? 

Feel free to close this "issue" if you feel it's outside of scope or not of interest pursuing.

**Comment by ncw** [comment]
I made http://rclone.org using this library - "rsync for cloud storage".  It doesn't have all the features of the official swift client, but it does a good job of the basics!  It is open source too.

**Comment by ncw** [comment]
I made http://rclone.org using this library - "rsync for cloud storage".  It doesn't have all the features of the official swift client, but it does a good job of the basics!  It is open source too.

**Comment by mlanner** [comment]
Yes, I'm well aware of Rclone. It's a great project. I'm using it regularly. I was more referring to something more akin to the Python Swift client. I'd love to use the Swift Go library to build a client, but my programming skills are not what they should be. So for me to get anywhere anytime soon is a long shot at best. But I really like the cross platform nature of Go/Rclone.
